manifests/repo.pp

```diff
@@ -64,7 +65,7 @@
         $key_server = 'hkp://keyserver.ubuntu.com:80'
       }
 
-      contain mongodb::repo::apt
+      contain ::mongodb::repo::apt
```

Comment:
please don't add the leading `::`. This is a deprecated syntax, required for Puppet 3.